src/main/java/org/spongepowered/plugin/meta/SpongeExtension.java

```diff
@@ -0,0 +1,15 @@
+package org.spongepowered.plugin.meta;
+
+public final class SpongeExtension {
+
+    public String assets; // public for mixin compatibility
+
+    public SpongeExtension(String assets) {
```

Comment:
It should never be null according to ModMetadata in Forge and there are instances where you might want to initialize it right away.